README.md

```diff
@@ -5,17 +5,21 @@ Maintain the source code for GraphQL related tools.
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![codecov](https://codecov.io/gh/ballerina-platform/graphql-tools/branch/main/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/graphql-tools)
- 
-Every GraphQL service defines a set of types which completely describe the set of possible data you can query on that service. Then, when queries come in, they are validated and executed against that schema. GraphQL schemas for a service are now most often specified using what’s known as the `GraphQL SDL` (schema definition language), also sometimes referred to as just GraphQL schema language. Ballerina GraphQL tooling will make it easy for you to start the development of a client in Ballerina for a given GraphQL SDL and a GraphQL document  configured in a GraphQL config file by generating Ballerina client skeletons. You can generate a single client in Ballerina for multiple GraphQL documents for a given GraphQL SDL. It also enables you to generate multiple Ballerina modules for multiple GraphQL projects to work with different GraphQL APIs. 
+
+Every GraphQL service defines a set of types that completely describe the set of possible data you can query on that service. Then, when queries come in, they are validated and executed against that schema. GraphQL schemas for a service are now most often specified using what’s known as the `GraphQL SDL` (schema definition language), also sometimes referred to as just GraphQL schema language. Ballerina GraphQL tooling will make it easy for you to start the development of a client in Ballerina for a given GraphQL SDL and a GraphQL document configured in a GraphQL config file by generating Ballerina client skeletons. You can generate a single client in Ballerina for multiple GraphQL documents for a given GraphQL SDL. It also enables you to generate multiple Ballerina modules for multiple GraphQL projects to work with different GraphQL APIs. In addition, the GraphQL tooling supports GraphQL SDL schema generation for a given Ballerina GraphQL service. The user will be able to generate the SDL schema and export it to a specific directory for a selected set of services in the given bal file.
 
 The Ballerina GraphQL tooling support provides the following capabilities.
- 
- 1. Generating a Ballerina client from a given GraphQL config file configured with a GraphQL schema (SDL) and a GraphQL document using the CLI tool.
 
- 2. Generating a Ballerina client from a given GraphQL config file configured with a GraphQL SDL and multiple GraphQL documents using the CLI tool.
+1. Generating a Ballerina client from a given GraphQL config file configured with a GraphQL schema (SDL) and a GraphQL document using the CLI tool.
+
+2. Generating a Ballerina client from a given GraphQL config file configured with a GraphQL SDL and multiple GraphQL documents using the CLI tool.
```

Comment:
Is the usage of `SDL` correct in these sentences? I don't think this is accurate. SDL means `Schema Definition Language`. Can we substitute this in the places where `SDL` is used and make meaningful sentence here? This applies multiple places, not just here. Shall we check and update them?

Reply:
Fixed in https://github.com/ballerina-platform/graphql-tools/pull/121/commits/33ace3fbc407810a3593b7b6758609bb9a608b42